metis/lib/models/file.rb

```diff
@@ -147,7 +148,11 @@ def self.hmac_url(method, host, path, expiration=0)
           expiration: (Time.now + expiration).iso8601,
           nonce: Metis.instance.sign.uid,
           id: :metis,
-          headers: { }
+          headers: user ? {
+            email: user.email,
+            first: user.first,
+            last: user.last
```

Comment:
Could we actually use `user.name` here instead? I've been meaning to deprecate First/Last for years.